inputs/demand/households/households_misc/households_number_of_residences.ad

```diff
@@ -35,10 +34,11 @@
       )
     )
   )
+
 - priority = 0
-- max_value_gql = present:AREA(number_of_residences)
-- min_value = 0.0
-- start_value_gql = present:AREA(number_of_semi_detached_houses)
+- max_value_gql = present:AREA(number_of_residences) * 1.0e-06 * 2
+- min_value_gql = present:AREA(number_of_residences) * 1.0e-06 * 0.5
+- start_value_gql = present:AREA(number_of_residences) * 1.0e-06
```

Comment:
I'm not sure if we should express the number of residences in millions. I think this works on country-level but I'm not sure if it works for local scenarios (provinces, municipalities, etc.) as well. Maybe we could express the number in thousands?

Reply:
If the unit is `mln`, I believe ETModel will scale it appropriately.

Reply:
Yes it works for me locally! :)

[Screen Shot 2019-06-27 at 14 29 57](https://user-images.githubusercontent.com/32833996/60266298-1265b200-98e8-11e9-9f9b-f0348ca4ef82.png)